src/SlackConnector/Connections/Clients/Chat/FlurlChatClient.cs

```diff
@@ -13,13 +13,14 @@ internal class FlurlChatClient : IChatClient
     {
         private readonly IResponseVerifier _responseVerifier;
         internal const string SEND_MESSAGE_PATH = "/api/chat.postMessage";
+        internal const string DELETE_MESSAGE_PATH = "/api/chat.delete";
 
         public FlurlChatClient(IResponseVerifier responseVerifier)
         {
             _responseVerifier = responseVerifier;
         }
 
-        public async Task PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
+        public async Task<PostMessageResponse> PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
```

Comment:
Hmm okay, I must be missing something because the Result is always null and the existing `PostMessage` call in `then_should_call_messenger` never gets called either. I've rarely written unit tests so that probably doesn't help my cause either :/

The naming convention for the function names is doing my head in too :(